build/gen_stub.php

```diff
@@ -1117,7 +1117,8 @@ private function initializeValue(string $type, $value, bool $isTyped): string
                 if (empty($value)) {
```

Comment:
I've just fixed this issue, and reviewed the actual property types. Basically all the integer properties can also be string because a string representation is returned in case of a possible overflow.